stubs/pika/pika/channel.pyi

```diff
@@ -38,7 +40,7 @@ class Channel:
         exclusive: bool = ...,
         consumer_tag: str | None = ...,
         arguments: _ArgumentMapping | None = ...,
-        callback: Callable[[Method], object] | None = ...,
+        callback: Callable[[Method[Queue.DeclareOk]], object] | None = ...,
```

Comment:
Should this be `ConsumeOk`? https://github.com/pika/pika/blob/22b381efa25b0b8671badcb7bb4485a7b2cefaca/pika/channel.py#L298-L299